network-config-analyzer/K8sServiceYamlParser.py

```diff
@@ -21,14 +21,17 @@ def __init__(self, service_file_name=''):
         """
         GenericYamlParser.__init__(self, service_file_name)
 
-    def parse_service(self, srv_object):
+    def parse_service(self, srv_object):  # noqa: C901
         """
         Parses a service resource object and creates a K8sService object
         :param dict srv_object: the service object to parse
         :return: K8sService object or None
         """
-        if srv_object.get('kind') != 'Service' or srv_object.get('apiVersion') != 'v1':
-            return None  # Not a v1 Service object
+        if srv_object.get('kind') != 'Service':
+            return None  # Not a Service object
+        api_version = srv_object.get('apiVersion')
+        if api_version != 'v1':
+            return None  # apiVersion is not properly set
```

Comment:
```suggestion
if srv_object.get('kind') != 'Service' or srv_object.get('apiVersion') != 'v1':
            return None  # Not a v1 Service object
```

Reply:
Was this change made due to a merge issue?

Reply:
Done